src/views/ipv/pyi-triage-select-device.njk

```diff
@@ -11,7 +11,7 @@
 
 {% block content %}
   <h1 class="govuk-heading-l" id="header" data-page="{{ googleTagManagerPageId }}">{{ 'pages.pyiTriageSelectDevice.header' | translate }}</h1>
-  <p class="govuk-body">{{ 'pages.pyiTriageSelectDevice.content.paragraph1' | translate }}</p>
+  <p class="govuk-hint">{{ 'pages.pyiTriageSelectDevice.content.hint' | translate }}</p>
```

Comment:
There's a hint parameter you can set in radiosConfig rather than creating a paragraph.

As a general rule whenever you're creating some HTML on page ask the question whether it's already done for you in a template :)

e.g. to find out about the hint go to this page: https://design-system.service.gov.uk/components/radios/ click on `Nunjucks` and then click on `Nunjucks macro options`

Reply:
This is a helpful reference I haven't seen before - I'll use this to also replace my hint implementation. On the replacement of the `h1` tag, I can't see how the legend would provide the `data-page` attribute, so I will leave it as it is for now